frontend/src/pages/signin-page.tsx

```diff
@@ -84,15 +88,58 @@ function signin() {
             )}
           </div>
 
-          <div className="mb-4">
+          <div className="relative mb-4">
             <input
               {...register('password')}
-              type="password"
+              type={showPassword ? 'text' : 'password'}
               placeholder="Password"
               className="w-full rounded-lg bg-zinc-100 p-3 font-normal placeholder:text-sm placeholder:text-neutral-500"
             />
+            <button
+              type="button"
+              className="absolute right-3 top-1/2 -translate-y-1/2 transform focus:outline-none"
+              onClick={togglePasswordVisibility}
+            >
+              {showPassword ? (
+                <svg
+                  xmlns="http://www.w3.org/2000/svg"
```

Comment:
kindly take this svg out and put it in assets and import like the other svgs being used in the project